Library/Homebrew/utils/install.rb

```diff
@@ -0,0 +1,41 @@
+# typed: false
+# frozen_string_literal: true
+
+require "utils/topological_hash"
+
+module Utils
+  # Helper module for installing formulae/casks.
+  #
+  # @api private
```

Comment:
Would be nice to make this a public API so it can be used by Homebrew/bundle: https://github.com/Homebrew/homebrew-bundle/blob/b8b5bdb46cc45b7be5f305654a145473469d19d0/lib/bundle/brew_dumper.rb#L177-L229